rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -111,7 +111,7 @@ def get_facets_sql(self, facets, table_prefix='ps'):
       for q_filter in filters:
         if str(q_filter) != '':
           filter_sql = filter_prefix + '.' + db_field + ' = ' + str(int(q_filter))
-          if allow_null:
+          if allow_null and str(int(q_filter)) == '1':
```

Comment:
Rather than do this, I'd add the "str(int(q_filter)) == '1'" at line 96 and leave this alone.

Reply:
Please ignore. :)

Reply:
As discussed on Slack, q_filter isn’t defined up there, only down where I’ve used it.  We agree this works.